operator/apis/mlops/v1alpha1/experiment_types.go

```diff
@@ -54,6 +54,11 @@ type ExperimentStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 //+kubebuilder:resource:shortName=mlx
+//+kubebuilder:printcolumn:name="Experiment ready",type=string,JSONPath=`.status.conditions[?(@.type=='ExperimentReady')].status`,description="Experiment ready status"
```

Comment:
how can we maintain these settings, eg. if we change some of these conditions that we are returning?

Reply:
They are quite difficult to directly test, mostly because this is an added comment used by kubebuilder when generating the CRDs (yaml files), rather than executable code. The failure mode here is quite lenient though, with any printcolumn referencing nonexistent data showing as empty.

Thinking outside the box a bit, we could imagine a test using the exact same fields as the kubebuilder comments, which will start failing when we change any of the status conditions. The failing test should contain a message explaining that one needs to change the comments here, and only update the test itself afterwards.

Reply:
I have accepted the challenge :-) and added tests. I might have exaggerated a bit, actually.

Reply:
thanks for finding a way to test it.